addons/core/functions/fnc_isPrototypeRadio.sqf

```diff
@@ -22,16 +22,12 @@
 
 if (_this == "ItemRadio") exitWith {true};
 
-
-//This is the same caching as in fnc_getConfigProperty. But we don't need that special stuff for CfgVehicles in getConfigProperty
-//So we implement the same caching for this more slim function.
-
-private _cacheName = (_this+"tf_prototype");
-private _cachedEntry = TFAR_ConfigCacheNamespace getVariable _cacheName;
+private _cacheName = (_this+"tf_prototypebool");
+private _cachedEntry = TFAR_ConfigWeaponCacheNamespace getVariable _cacheName;
```

Comment:
Why do you have a copy of the cache function around the cached function here?

Reply:
because this way, i can return a bool without checking a number against a bool.